dataset.py

```diff
@@ -362,11 +362,21 @@ def _get_single_item(self, idx, image_height):
         image = self._load_image(idx)
 
         # Load intrinsics.
-        focal_length = float(np.loadtxt(self.calibration_files[idx]))
+        k = np.loadtxt(self.calibration_files[idx])
+        if k.size == 1:
+            focal_length = float(k)
+        else:
+            k = k.tolist()
+            focal_length = [k[0][0], k[1][1]]
+            centre_point = [k[0][2], k[1][2]]
```

Comment:
@tcavallari we can change this to just be 4 values or whatever you think is more appropriate. Currently we assume that that it's a 3x3 intrinsics matrix like so: 

```
fx 0 cx
0 fy cy
0 0 1
```

just for the sake of being consistent with literature notation

Reply:
3x3 matrix is fine. Can you add a check to make sure that's the actual shape of the input matrix?

Reply:
Thanks!